src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -0,0 +1,333 @@
+import moment from 'moment';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  LineChart, Line, BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer,
+} from 'recharts';
+
+import Spinner from '../../components/Spinner';
+import { numberWithCommas, downloadFromGuppy } from '../dataUtils.js';
+import './PlotChart.less';
+
+
+class PlotChartAxisTick extends React.Component {
+  /* eslint-disable react/prop-types */
```

Comment:
why this rule is disabled in here?

Reply:
because `PlotChartAxisTick` contains `const { x, y, payload, axis, type } = this.props;` and then `payload.value` so eslint complains the prop types are not described

`payload` comes from `XAxis.tick` and `YAxis.tick` and i'm not sure what else it contains

tldr: laziness 🤔

Reply:
oh i'd like we put them into the `propTypes` but i also understand the tldr part